appveyor/install-miniconda.ps1

```diff
@@ -76,8 +76,12 @@ $env:PATH = "${env:PYTHON};${env:PYTHON}\Scripts;" + $env:PATH
 
 # Conda config
 conda config --set always_yes true
-conda config --add channels astropy-ci-extras
 
+if ($conda_channels -ne $null) {

```

Comment:
please use upper case global variable names. Also ``-ne $null`` is not needed I think, if a variable is set it should be ``True`` in the conditional (see examples below for ``ASTROPY_VERSION``).